addons/web/controllers/main.py

```diff
@@ -491,14 +491,22 @@ def _remove_inheritance_comments(self, inherited_template):
             if re.match(COMMENT_PATTERN, comment.text.strip()):
                 comment.getparent().remove(comment)
 
-    def _read_addon_file(self, file_path):
-        """Reads the content of a file given by file_path
-        Usefull to make 'self' testable
-        :param str file_path:
+    def _read_addon_file(self, path_or_url):
+        """Read the content of a file or an 'ir.attachment' record given by
+        'path_or_url'. A 'FileNotFoundError' is raised if the path doesn't match
```

Comment:
```suggestion
        """Read the content of a file or an ``r.attachment`` record given by
        ``path_or_url``. A :exc:`FileNotFoundError` is raised if the path doesn't match
```
Alternatively, the description of `FileNotFoundError` can be moved to a `raises` info field e.g.
```
:raises FileNotFoundError: if the path does not match a module file or an attachment
```